appveyor.yml

```diff
@@ -0,0 +1,49 @@
+version: 1.0.0_b{build}-{branch}
+
+os: Visual Studio 2015
+
+environment:
+  TERM: dumb
+  COVERALLS_REPO_TOKEN:
+    secure: 5nuCg+faxFPeppoNNcSwVobswAVFUf8ut83vw8CX/4W2y0kZkGmwEfCUxSQWiQDU
+  CI_NAME: appveyor
+  CI_JOB_ID: $(APPVEYOR_JOB_ID)
+  GCOV_PREFIX: $(APPVEYOR_BUILD_FOLDER)
+  matrix:
+  - COMPILER: msvc
+    GENERATOR: "Visual Studio 14 2015"
+    CFLAGS: /MP
+    CXXFLAGS: /MP
+  - COMPILER: mingw
+    GENERATOR: "MSYS Makefiles"
+
+platform:
+  - x86
+  - x64
+
+configuration:
+  - Debug
+  - Release
+  - RelWithDebInfo
+
+init:
+  - set PATH=C:\msys64\mingw64\bin;C:\msys64\usr\bin;C:\Python35;C:\Python35\Scripts;%PATH%
+  - set MSYSTEM=MINGW64
+  - pip install cram==0.7
+
+install:
+  - mkdir build && cd build
+  - >
+      cmake
+      -Wno-dev
+      -DCMAKE_INSTALL_PREFIX=boxfort-%RELEASE_NAME%
+      -DCMAKE_BUILD_TYPE="%CONFIGURATION%"
+      -DBXF_STATIC_LIB=ON ..
```

Comment:
I set the PATH, but apparently it is not looking for `boxfort.dll` but complaining about `boxfort.lib` https://ci.appveyor.com/project/am11/boxfort/build/1.0.0_b12-build/vcr.

So I reverted this line to explicitly build as static lib (fixed the typo; removed `..` from this line). Now the library is build: https://ci.appveyor.com/project/am11/boxfort/build/1.0.0_b13-build/vcr (failing unit tests though)

Maybe we need to have (a simplified version of) what's there in Criterion:
https://github.com/Snaipe/Criterion/blob/4d6d741/.cmake/Modules/PackageUtils.cmake#L28-L57